tests/utils_test.py

```diff
@@ -106,7 +106,11 @@ def sort_nested(data):
         new_data = {}
         for k, v in data.items():
             if isinstance(v, list):
-                v = sorted(v)
+                try:
+                    v = sorted(v)
```

Comment:
This is the case where `data` is a dict and `data[k]` is of type `list`. Why isn't `sort_nested` called recursively here? In the situation where `data` is a list and a given value `v` is of type `list`, `sort_nested` is called on it.

Reply:
You're correct, it's appropriate to call sort_nested() here.

Reply:
Cool, thanks for coming back so quickly!